.travis.yml

```diff
@@ -1,8 +1,20 @@
-language: node_js
 sudo: false
+
+git:
+  depth: 1
+
+branches:
+  only:
+    - master
```

Comment:
I wondered the same thing too. The easiest way to find out is to fail a test in this branch, which I'll do now.